src/csharp.tmLanguage.yml

```diff
@@ -2360,6 +2362,9 @@ repository:
   ref-modifier:
     name: storage.modifier.cs
     match: \b(ref)\b
+  readonly-modifier:
```

Comment:
could you add a line break before this new rule to be consistent with the rest of the grammar?